editor/editor_help.cpp

```diff
@@ -3868,6 +3869,37 @@ void EditorHelpBit::_update_labels() {
 		_add_text_to_rt(help_data.description.replace("<EditorHelpBitCommentColor>", comment_color.to_html()), content, this, symbol_class_name);
 	}
 
+	if (!help_data.resource_path.is_empty()) {
+		if (has_prev_text) {
+			content->add_newline();
+			content->add_newline();
+		}
+		has_prev_text = true;
+
+		const String ext = help_data.resource_path.get_extension();
+		const bool is_dir = ext.is_empty();
+		const bool is_valid = is_dir || EditorFileSystem::get_singleton()->get_valid_extensions().has(ext);
+		if (!is_dir && is_valid) {
+			content->push_meta("open " + help_data.resource_path, RichTextLabel::META_UNDERLINE_ON_HOVER);
+			content->add_image(get_editor_theme_icon(SNAME("Load")));
+			content->add_text(" " + TTR("Open"));
```

Comment:
```suggestion
			content->add_image(get_editor_theme_icon(SNAME("Load")));
			content->add_text(nbsp + TTR("Open"));
			content->push_meta("open " + help_data.resource_path, RichTextLabel::META_UNDERLINE_ON_HOVER);
```

1. Use a non-breaking space.
2. Move the icon outside the link so it is not underlined.

The same below two more times.

Reply:
While it might look slightly strange that the icon gets underlined, I still think it's better this way since it gives you more "hit area" to click the link, and it mimics context menus more this way. It feels weird to not be able to click the icon. I did add the `nbsp` though.